src/popup/router/components/AccountCard.vue

```diff
@@ -28,8 +28,8 @@ export default {
   },
   props: {
     idx: { type: Number, required: true },
-    color: { type: String, required: true },
-    shift: { type: Number, required: true },
+    color: { type: String },
```

Comment:
The `color` and `shift` is actually required, why do you decide to change that.

Reply:
Because when `AccountCard` is used somewhere it always receive all of the props through `v-bind` and it resulted in errors in the tests